grc/utils/form_custom_validators.py

```diff
@@ -116,12 +116,6 @@ def validateReferenceNumber(form, field):
         raise ValidationError('Enter a valid reference number')
 
 
-def validateEmailAddress(form, field):
-    from email.utils import parseaddr
-    if not '@' in parseaddr(field.data)[1]:
-        raise ValidationError('Enter a valid email address')
-
-
```

Comment:
This code isn't used any more.
We seem to be using the `wtforms.validators.Email` validator instead
e.g. https://github.com/cabinetoffice/grc-app/blob/master/admin/admin/forms.py#L10